File: JunsBlog/ClientApp/src/app/models/commentDetails.ts
import { User } from './user';
import { CommentRankingDetails } from './commentRankingDetails';

export class CommentDetails{
    ranking: CommentRankingDetails;
    user: User;
    commentText: string;
    updatedOn: string;
    content: string;
    articleId: string;
    id: string;
    parentId: string;
    childrenCommentsCount: number;
}

File: JunsBlog/Models/Comments/CommentDetails.cs
using JunsBlog.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace JunsBlog.Models.Comments
{
    public class CommentDetails
    {
        public string Id { get; set; }
        public DateTime UpdatedOn { get; set; }
        public string CommentText { get; set; }
        public User User { get; set; }
        public int ChildrenCommentsCount { get; set; }
        public string ArticleId { get; set; }
        public string ParentId { get; set; }
        public CommentRankingDetails Ranking { get; set; }

    }
}
